### ISSUE TITLE
Empty ruleset created when containing child + comment

### ISSUE THREAD

**Issue by simonsmith** [postcss/postcss-nested#19]
It seems that if you use comments above child rulesets then they exist in the parent, creating an empty ruleset. 

**Input**

```css
.Parent {
  /* comment about Child */
  .Child {
    width: auto;
  }

  /* another comment about this one */
  .AnotherChild {
    width: auto;
  }
}
```

**Expected**

``` css
/* comment about Child */
.Parent .Child {
  width: auto;
}

/* another comment about this one */
.Parent .AnotherChild {
  width: auto;
}
```

** Actual **

``` css
.Parent {
  /* comment about Child */

  /* another comment about this one */
}

.Parent .Child {
  width: auto;
}

.Parent .AnotherChild {
  width: auto;
}
```

It's interesting to note that Sass and Less also have this issue, but Stylus correctly matches the expected output however it does remove the comments. 

I'm guessing they get round this issue by encouraging `//` style comments.

Wondering what should happen in this case.

**Comment by ai** [comment]
Done e709092

**Comment by simonsmith** [comment]
:+1:

**Comment by ai** [comment]
Released in 1.0
